app/assets/stylesheets/colors.scss

```diff
@@ -29,6 +29,8 @@ $sidebars-background: #f0f0f0;
 $sidebars-sub-background: darken($sidebars-background, 12%);
 $left-navbar-drawer-background: darken($sidebars-background, 6%);
 
+$card-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
```

Comment:
A new `variables.scss` file. Although I'm not sure if we really need this variable here. Instead you could just add the value itself in `_flash_messages.scss`.